pandas/tests/io/formats/test_to_html.py

```diff
@@ -415,6 +431,85 @@ def test_to_html_multiindex_max_cols(self, datapath):
         expected = expected_html(datapath, 'gh6131_expected_output')
         assert result == expected
 
+    def test_to_html_multi_indexes_index_false(self, datapath):
+        # GH 22579
+        df = DataFrame({'a': range(10), 'b': range(10, 20), 'c': range(10, 20),
+                        'd': range(10, 20)})
+        df.columns = MultiIndex.from_product([['a', 'b'], ['c', 'd']])
+        df.index = MultiIndex.from_product([['a', 'b'],
+                                            ['c', 'd', 'e', 'f', 'g']])
+        result = df.to_html(index=False)
+        expected = expected_html(datapath, 'gh22579_expected_output')
+        assert result == expected
+
+    @pytest.mark.parametrize(
+        ('idx_flag', 'named_index', 'expected_output'),
```

Comment:
Can you make the first argument to `parametrize` a single string? I haven't seen a Tuple passed like this before in other parts of the codebase nor in their documentation